truffle-config.js

```diff
@@ -1,26 +1,73 @@
 const path = require("path");
+const fs = require('fs');
 const HDWalletProvider = require('@truffle/hdwallet-provider');
 
 //Put your mnemonic here, take care of this and don't deploy your mnemonic into production!
 const mnemonic = 'A_MNEMONIC';
 
+//Update gas price Testnet
+/* Run this first, to use the result in truffle-config:
+  curl https://public-node.testnet.rsk.co/ -X POST -H "Content-Type: application/json" \
+    --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["latest",false],"id":1}' \
+    > .minimum-gas-price-testnet.json
+*/
+const gasPriceTestnetRaw = fs.readFileSync(".minimum-gas-price-testnet.json").toString().trim();
+const minimumGasPriceTestnet = parseInt(JSON.parse(gasPriceTestnetRaw).result.minimumGasPrice, 16);
+if (typeof minimumGasPriceTestnet !== 'number' || isNaN(minimumGasPriceTestnet)) {
+  throw new Error('unable to retrieve network gas price from .gas-price-testnet.json');
+}
+console.log("Minimum gas price Testnet: " + minimumGasPriceTestnet);
+
```

Comment:
Great idea!

Reply:
This was Brendan idea too :)

Reply:
We are using in other tutorials, I will update all Truffle boxes to use it